try: {
    1 / 0
except Exception as e:
    var trace = e.traceback[0]
    var path = trace[0]
    var func = trace[1]
    var line = trace[2]
    if path.endswith("validate_traceback.lly") == false ||
       func != "__main__" ||
       line != 2:
        stderr.write("Failed.\n")
}
